File: 001/1.c
// Add all the natural numbers below one thousand that are multiples of 3 or 5.

#include <stdio.h>

int main () {
	int sum = 0;
	int i = 0;
	while (i < 1000) {
		if (i % 3 == 0 || i % 5 == 0) {
			if (i % 3 == 0 && i % 5 == 0) {
				sum += i;
			} else {
				sum += i;
			}
		}
		i++;
		printf("Summe ist: %i\n", sum);
	}
}

File: 001/1.swift
// If we list all the natural numbers below 10 that are multiples of 3 or 5,
// we get 3, 5, 6 and 9. The sum of these multiples is 23.
//
// Find the sum of all the multiples of 3 or 5 below 1000.

var sum = 0

for i in 0..<1000 {
    if i % 3 == 0 || i % 5 == 0 {
        sum += i
    }
}

print(sum)
